clap_builder/src/builder/command.rs

```diff
@@ -2295,6 +2300,22 @@ impl Command {
         self
     }
 
+    /// Set the default section heading for future subcommands.
+    ///
+    /// This will be used for any subcommand that hasn't had [`Command::subcommand_help_heading`] called.
+    ///
+    /// This is useful if the default `Commands` heading is
+    /// not specific enough for one's use case.
+    ///
+    /// [`Command::subcommand`]: Command::subcommand()
+    /// [`Command::subcommand_help_heading`]: crate::Command::subcommand_help_heading()
+    #[inline]
+    #[must_use]
+    pub fn next_subcommand_help_heading(mut self, heading: impl IntoResettable<Str>) -> Self {
```

Comment:
Should we only have one `next_help_heading`?

Reply:
Good question, that would make api simpler, but I'd rather have clear separation between subcommand and parameter heading

Reply:
Could you explain why?